Backend_NodeJS/src/config/index.js

```diff
@@ -3,7 +3,6 @@ config();
 const { PORT } = process.env;
 const port = PORT || 8080;
 const dbUri =
-  // "mongodb+srv://admin:admin@revive.p6okv5v.mongodb.net/?retryWrites=true&w=majority";
   "mongodb+srv://admin:admin@projectgallery.rbwfrho.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
@SriHarikaV We don't need these changes to be pushed every time.

Reply:
@anjaliputta Unfortunately, it appeared because before I updated on the main branch rather than my branch. This won't happen again.